### DISCUSSION TITLE
How to add startup parameters to all CefSharp.BrowserSubprocess?

### DISCUSSION THREAD

**Discussion by shouhujishu** [cefsharp/CefSharp#4283] [Q&A] [answered]
I am rewriting libcef.dll to accept startup commands, but CefSharp.BrowserSubprocess multi-process causes me to fail to receive commands, how can I implement parameter startup?

**Comment by amaitland** [comment, marked as answer]
Command line args added programatically are sent to the sub process. 

Look at the example code in https://github.com/cefsharp/CefSharp/wiki/General-Usage#cefsettings-and-browsersettings

**Reply by shouhujishu** [reply]
> Command line args added programatically are sent to the sub process.
> 
> Look at the example code in https://github.com/cefsharp/CefSharp/wiki/General-Usage#cefsettings-and-browsersettings
Hello, this method is limited to the main process, the Sub process will not attach custom startup parameters.

**Reply by amaitland** [reply]
Chromium will propagate the command line args that are relevant to the sub process. There's no option for **custom** args currently. 

You'll need to modify https://github.com/cefsharp/CefSharp/blob/master/CefSharp.Core.Runtime/Internals/CefSharpApp.h#L124
